packages/build/src/commands/plugin.js

```diff
@@ -43,6 +50,37 @@ const firePluginCommand = async function ({
       location: { event, packageName, loadedFrom, origin },
     })
     return { newError }
+  }
+}
+
+const callSameProcess = function ({ context, event, loadedFrom, buildDir, constants, events, error, logs }) {
+  return run(
+    { event, events, error, envChanges: {}, constants: { ...constants, BUILD_DIR: buildDir }, loadedFrom, logs },
```

Comment:
`envChanges` are `process.env` modifications from previous plugins. We do not want to modify the parent process `env`, so we pass an empty object.

We also pass `BUILD_DIR`. In child process, this is the current directory. Since we use the same process, we pass it as a constant instead. We could also `process.chdir()`, but we might want to avoid global state mutation if possible.

Finally, we pass `logs`. Logs in Neltify Build are either printed on the console or stored in an array, depending on the `buffer` option. This is especially useful during tests. In child process, we fetch the child process' `stdout`. Since we use the same process here, we need to be more explicit.